lectures/pv.md

```diff
@@ -72,9 +72,9 @@ We say equation**s**, plural, because there are $T+1$ equations, one for each $t
 
 Equations {eq}`eq:Euler1` assert that price paid to purchase  the asset at time $t$  equals the payout $d_t$  plus the price at time  $t+1$ multiplied by a time discount factor $\delta$.
 
-Discounting tomorrow's price  by multiplying it by  $\delta$ accounts for the ``value of waiting one period''.
+Discounting tomorrow's price  by multiplying it by  $\delta$ accounts for the ''value of waiting one period''.
```

Comment:
@SylviaZhaooo it would be great if you could update these to `"` rather than double ticks.